### DISCUSSION TITLE
Red or green icon

### DISCUSSION THREAD

**Discussion by Hugh-1901** [kkapsner/CanvasBlocker#727] [Q&A] [answered]
It's not obvious to me what the colour of the icon means. Does a green thumb print mean Canvas Blocker is enabled? Red is disabled?
Or the other way around? Or something else entirely?

Can't find it in what docs there are.

**Comment by spodermenpls** [comment, marked as answer]
CanvasBlocker is enabled in both cases. A green icon means "no fingerprintable API had to be protected by CanvasBlocker on this website, so far", a red icon means "at least one fingerprintable API has been protected by CanvasBlocker on this website". A disabled CanvasBlocker would get visualized with a gray icon.

**Reply by Hugh-1901** [reply]
Thanks!

**Reply by Hugh-1901** [reply]
I notice that github.com has a red icon, LOL

**Reply by spodermenpls** [reply]
> I notice that github.com has a red icon, LOL

You have to keep in mind, all of the APIs that CanvasBlocker protects have legitimate usecases, that they're actually used for fingerprinting is the exception, not the norm.

**Comment by kkapsner** [comment]
If you hover over the icon you will get a tooltip with some information what API was used:
![grafik](https://github.com/user-attachments/assets/0e0631e9-309a-4b85-a5c6-9271978587eb)
